src/containers/Nodes/Nodes.tsx

```diff
@@ -90,15 +86,14 @@ export const Nodes = ({path, type, additionalNodesProps = {}}: NodesProps) => {
                 dispatch(setDataWasNotLoaded());
             }
 
-            // For not DB entities we always use /compute endpoint instead of /nodes
-            // since /nodes can return data only for tenants
-            if (path && (!useNodesEndpoint || !isDatabaseEntityType(type))) {
+            // If there is no path, it's cluster Nodes tab
+            if (path && !useNodesEndpoint) {
                 dispatch(getComputeNodes({path}));
             } else {
-                dispatch(getNodes({tenant: path}));
+                dispatch(getNodes({path}));
             }
         },
```

Comment:
1. Use `/nodes` instead of `/compute` for not DB entities in Diagnostics
2. Request entities nodes by `path` param instead of `tenant`